src/providers/boleto-api-caixa/index.js

```diff
@@ -114,6 +116,38 @@ const defineInterest = (boleto) => {
   return null
 }
 
+const buyerAdressIsEqualsToPagarme = (boleto) => {
```

Comment:
opa, @davidsmedeiros aqui da para retornarmos direto o resultado 😀 


```suggestion
const buyerAdressIsEqualsToPagarme = (boleto) => {
```
```suggestion
const buyerAdressIsEqualsToPagarme = (boleto) => (equals(boleto.payer_address, boleto.company_address))
```